alert.js

```diff
@@ -1,282 +1,103 @@
-function min(a, b)
-{
-    return (a > b) ? alert(b): alert(a);
-}
-min(a,b);*/
-
-//3.1-3.6
-/*let x = prompt("Введите значения x: ", "0");
-let n = prompt("Введите значения n: ", "0");
-function pow(x, n) {
-    let res = Math.pow(x, n);
-    alert(res);
-}
+    },
 
-pow(x, n);
+    mul() {
+        let mul = this.val1 * this.val2;
+        return mul;
 
-let res = confirm("Вы согласны?") ? () => alert("Вы согласились."): () => alert("Вы отменили выполнение.");
-res();
-
-function pow(x,n) {
-    let result = 1;
-
-    for ( let i = 0; i < n; i++) {
-
-        result*=x;
     }
-
-    return result;
-}
-
-let x = prompt("x?",'');
-let n=prompt("n?",'');
-
-if (n<=0) {
-    alert(`Степень ${n} не поддерживается,
-           введите целую степень, большую 0`);
-} else {
-    alert(pow(x, n));
-}*/
-
-/*let user = {
-    name: "John",
-    surname: "Smith"
 };
 
-alert(user.name);
-
-user.name = "Pete";
-
-alert(user.name);
-
-delete user.name;
-
-let schedule = {};
+calculator.read();
+alert( calculator.sum() );
+alert( calculator.mul() );
+
+let ladder = {
+    step: 0,
+    up() {
+        this.step++;
+        return this;
+    },
+    down() {
+        this.step--;
+        return this;
+    },
+    showStep: function() { // показывает текущую ступеньку
+        alert( this.step );
+        return this;
+    }
+};
 
+ladder.up().up().down().showStep();*/
 
-function isEmpty(obj) {
+/*let obj = {};
 
-    for (let key in obj)
-    {
-        return false;
-    }
-    return true;
-}
+function A() {return obj; }
 
-alert( isEmpty(schedule) );
+function B() {return obj; }
 
-schedule["8:30"] = "get up";
+let a = new A;
+let b = new B;
 
-alert( isEmpty(schedule) );
+alert( a == b );*/
 
-let salaries = {
-    John: 100,
-    Ann: 160,
-    Pete: 130
-}
+/*function Calculator() {
+    this.read = function () {
+        this.val1 = +prompt("Введите первое значение: ");
+        this.val2 = +prompt("Введите второе значение: ");
+    },
+    this.sum = function () {
+        let sum = this.val1 + this.val2;
+        return sum;
 
-function isSalaries(obj)
-{
-    let sum = 0;
+    },
 
-    for (let key in obj) {
-        sum += obj[key];
+    this.mul = function () {
+        let mul = this.val1 * this.val2;
+        return mul;
     }
-    return sum;
-}
 
-alert(isSalaries(salaries));*/
+}*/
+
+/*let calculator = new Calculator();
+calculator.read();
 
-let menu = {
-    author: "Oleg",
-    width: 200,
-    height: 300,
-    title: "My menu"
-};
+alert( "Sum=" + calculator.sum() );
+alert( "Mul=" + calculator.mul() );*/
 
-function multiplyNumeric(obj) {
+function Accumulator(startingValue) {
+    this.value = startingValue;
 
-    let key;
-    for (key in obj) {
-        if (typeof obj[key] == "number") {
-            obj[key] = 3 * obj[key];
-        }
+    this.read = function () {
+        this.number = +prompt("Введите число: ");
+        this.value += this.number;
+        console.log(this.value);
     }
 }
 
```

Comment:
create an function class (such as Accumulator) Soldier and create three methods shoot, heal, run 
create an condition if u see an enemy soldier have to shoot if soldier get injured heal and if they going too lose run